src/test/java/automationexercise/pageobjects/FormObjects.java

```diff
@@ -0,0 +1,34 @@
+package automationexercise.pageobjects;
+
+import automationexercise.hooks.Hooks;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+
+public class FormObjects extends BasePageObjects {
+
+    public FormObjects() {
+        super();
+    }
+
+    public void fillInputField(String selector, String value) {
+        WebElement inputField = Hooks.getWebDriver().findElement(new By.ByCssSelector(selector));
+        JavascriptExecutor js2 = (JavascriptExecutor) driver;
```

Comment:
Scrolling the element into view could go to a separate method to avoid code duplication